packages/http-client-csharp/emitter/src/type/input-decorated-type.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { DecoratorInfo } from "@azure-tools/typespec-client-generator-core";
+
+export interface InputDecoratedType {
+  Decorators?: DecoratorInfo[];
+}
```

Comment:
Could we only export this internally? I do not think external code would need this.

Reply:
Hello @ArcturusZhang `InputDecoratedType` will be used in more than one files in this library, so we export this internally. We don't export any input type externally.